### ISSUE TITLE
Links are capped

### ISSUE THREAD

**Issue by joas8211** [tahnik/devRantron#171]
I don't know if "capped" is the perfect word for it, but as link's text's size is limited is the URL itself also limited, so it no longer points to the correct place.

[Example rant](https://devrant.io/rants/781660)

In the example, link text https://www.youtube.com/watch?v=IBU1lTfZ4Co is capped to https://www.youtube.com/watch..., so devRantron uses the text also as a link to https://www.youtube.com/watch/, which is not what we want.

TL;DR
Is like this: [https://www.youtube.com/watch...](https://www.youtube.com/watch/)
Should be like this: [https://www.youtube.com/watch...](https://www.youtube.com/watch?v=IBU1lTfZ4Co)

**Comment by tahnik** [comment]
@joas8211 have you updated to the latest version? I recently fixed it. Please download the latest version and let me know if the problem still persists.

Here is the link: https://devrantron.firebaseapp.com/

**Comment by joas8211** [comment]
@tahnik It could be it, client says it is up to date, but I guess it cannot be trusted? I'm downloading latest version, wait a minute (slow internet).

**Comment by tahnik** [comment]
@joas8211 which operating system are you using?

**Comment by joas8211** [comment]
@tahnik Linux Mint 18.1

**Comment by joas8211** [comment]
@tahnik It's working in v1.3.3, so closed. But I'll look again the no-updates problem.

**Comment by tahnik** [comment]
Auto update doesn't work in Linux. It's not supported by electron yet. Sorry about that :(

**Comment by joas8211** [comment]
@tahnik How 'bout implementing Linux supported updater like [electron-simple-updater](https://github.com/megahertz/electron-simple-updater)?

**Comment by tahnik** [comment]
That package only supports AppImage updates in Linux. And as far as I know, our AppImage also supports updates, but not automatically.

We are planning to push this app to repositories soon anyway. So you should be able to update via apt, dnf etc...

**Comment by hex2f** [comment]
We're also adding a "What's new" popup to inform of update and which features it adds.
